tests/dragonfly/snapshot_test.py

```diff
@@ -61,8 +87,30 @@ async def test_snapshot(self, df_seeder_factory, async_client, df_server):
         # save + flush + load
         await async_client.execute_command("SAVE DF")
         assert await async_client.flushall()
-        await async_client.execute_command("DEBUG LOAD " + super().get_main_file("dfs"))
+        await async_client.execute_command("DEBUG LOAD " + super().get_main_file("dfs", "dfs"))
```

Comment:
please dont hide the "test-" prfix inside the get_main_file, it is a bit confusing